src/languages/singlestoredb/singlestoredb.formatter.ts

```diff
@@ -0,0 +1,267 @@
+  'ADD LEAF',
+  'AGGREGATOR SET AS MASTER',
+  'ALTER DATABASE',
+  'ALTER PIPELINE',
+  'ALTER RESOURCE POOL',
+  'ALTER USER',
+  'ALTER VIEW',
+  'ANALYZE TABLE',
+  'ATTACH DATABASE',
+  'ATTACH LEAF',
+  'ATTACH LEAF ALL',
+  'BACKUP DATABASE',
+  'BINLOG',
+  'BOOTSTRAP AGGREGATOR',
+  'CACHE INDEX',
+  'CALL',
+  'CHANGE',
+  'CHANGE MASTER TO',
+  'CHANGE REPLICATION FILTER',
+  'CHANGE REPLICATION SOURCE TO',
+  'CHECK BLOB CHECKSUM',
+  'CHECK TABLE',
+  'CHECKSUM TABLE',
+  'CLEAR ORPHAN DATABASES',
+  'CLONE',
+  'COMMIT',
+  'CREATE DATABASE',
+  'CREATE GROUP',
+  'CREATE INDEX',
+  'CREATE LINK',
+  'CREATE MILESTONE',
+  'CREATE PIPELINE',
+  'CREATE RESOURCE POOL',
+  'CREATE ROLE',
+  'CREATE USER',
+  'CREATE VIEW',
+  'DEALLOCATE PREPARE',
+  'DESCRIBE',
+  'DETACH DATABASE',
+  'DETACH PIPELINE',
+  'DO',
+  'DROP DATABASE',
+  'DROP FUNCTION',
+  'DROP INDEX',
+  'DROP LINK',
+  'DROP PIPELINE',
+  'DROP PROCEDURE',
+  'DROP RESOURCE POOL',
+  'DROP ROLE',
+  'DROP USER',
+  'DROP VIEW',
+  'EXECUTE',
+  'EXPLAIN',
+  'FLUSH',
+  'FORCE',
+  'GRANT',
+  'HANDLER',
+  'HELP',
+  'KILL CONNECTION',
+  'KILLALL QUERIES',
+  'LOAD DATA',
+  'LOAD INDEX INTO CACHE',
+  'LOAD XML',
+  'LOCK INSTANCE FOR BACKUP',
+  'LOCK TABLES',
+  'MASTER_POS_WAIT',
+  'OPTIMIZE TABLE',
+  'PREPARE',
+  'PURGE BINARY LOGS',
+  'REBALANCE PARTITIONS',
+  'RELEASE SAVEPOINT',
+  'REMOVE AGGREGATOR',
+  'REMOVE LEAF',
+  'REPAIR TABLE',
+  'REPLACE',
+  'REPLICATE DATABASE',
+  'RESET',
+  'RESET MASTER',
+  'RESET PERSIST',
+  'RESET REPLICA',
+  'RESET SLAVE',
+  'RESTART',
+  'RESTORE DATABASE',
+  'RESTORE REDUNDANCY',
+  'REVOKE',
+  'ROLLBACK',
+  'ROLLBACK TO SAVEPOINT',
+  'SAVEPOINT',
+  'SET CHARACTER SET',
+  'SET DEFAULT ROLE',
+  'SET NAMES',
+  'SET PASSWORD',
+  'SET RESOURCE GROUP',
+  'SET ROLE',
+  'SET TRANSACTION',
+  'SHOW',
+  'SHOW CHARACTER SET',
+  'SHOW COLLATION',
+  'SHOW COLUMNS',
+  'SHOW CREATE DATABASE',
+  'SHOW CREATE FUNCTION',
+  'SHOW CREATE PIPELINE',
+  'SHOW CREATE PROCEDURE',
+  'SHOW CREATE TABLE',
+  'SHOW CREATE USER',
+  'SHOW CREATE VIEW',
+  'SHOW DATABASES',
+  'SHOW ENGINE',
+  'SHOW ENGINES',
+  'SHOW ERRORS',
+  'SHOW FUNCTION CODE',
+  'SHOW FUNCTION STATUS',
+  'SHOW GRANTS',
+  'SHOW INDEX',
+  'SHOW MASTER STATUS',
+  'SHOW OPEN TABLES',
+  'SHOW PLUGINS',
+  'SHOW PRIVILEGES',
+  'SHOW PROCEDURE CODE',
+  'SHOW PROCEDURE STATUS',
+  'SHOW PROCESSLIST',
+  'SHOW PROFILE',
+  'SHOW PROFILES',
+  'SHOW RELAYLOG EVENTS',
+  'SHOW REPLICA STATUS',
+  'SHOW REPLICAS',
+  'SHOW SLAVE',
+  'SHOW SLAVE HOSTS',
+  'SHOW STATUS',
+  'SHOW TABLE STATUS',
+  'SHOW TABLES',
+  'SHOW VARIABLES',
+  'SHOW WARNINGS',
+  'SHUTDOWN',
+  'SNAPSHOT DATABASE',
+  'SOURCE_POS_WAIT',
+  'START GROUP_REPLICATION',
+  'START PIPELINE',
+  'START REPLICA',
+  'START SLAVE',
+  'START TRANSACTION',
+  'STOP GROUP_REPLICATION',
+  'STOP PIPELINE',
+  'STOP REPLICA',
+  'STOP REPLICATING',
+  'STOP SLAVE',
+  'TABLE',
```

Comment:
I suspect there's no `TABLE`-statement or `TABLE`-clause within another statement.

I know MySQL also lists `TABLE` here, which is also wrong. There's unfortunately quite a bit of such mistakes in the existing `reservedCommands` lists for all dialects. I'm currently in the process of reviewing these lists and throwing out incorrect stuff. Only the top of the list (until an empty line) is the reviewed part, that I'm more certain of. The rest should be taken with grain of salt, and better not to be copied over without checking.